src/ExportMap.js

```diff
@@ -262,7 +262,7 @@ function captureTomDoc(comments) {
 }
 
 ExportMap.get = function (source, context) {
-  const path = resolve(source, context)
+  const path = resolve(source, context, __dirname)
```

Comment:
this would be the dirname of this file - is that intended? I'd expect `process.cwd()`

Reply:
This is where we load the "resolver module" from, not the actual module that we're resolving. I've updated the doc comments.